docs/creating-a-kubernetes-cluster.md

```diff
@@ -28,21 +28,20 @@ gcloud container clusters get-credentials --zone us-east1-d knative-demo
 
```

Comment:
I have a change here: https://github.com/knative/serving/pull/2962 that removes this file as it is duplicated by the platform specific creation guides found here: https://github.com/knative/docs/tree/master/install